dist/elastio-snap.spec

```diff
@@ -220,6 +220,13 @@ Group:           System Environment/Kernel
 BuildArch:       noarch
 %endif
 
+%if 0%{?debian}
+%if ( "%{_arch}" != "x86_64" && "%{_arch}" != "amd64" ) && ( %{debian} == 11 )
+# Please refer to https://github.com/elastio/devboxes/pull/230
+Requires:        linux-image-%(uname -r)-dbg
+%endif
```

Comment:
It would be great to say something about that the kernels on Debian 11 are compiled without `CONFIG_KALLSYMS_ALL` and about possibility to find `sys_call_table` etc. All of this your investigations are not reflected neither in the PR's description nor in the issue. (It wasn't known when the issue #124 was created)

Reply:
Yes agreed, that knowledge should be captured somewhere in the code lest we forget it.

Reply:
Done